code/game/objects/items/devices/megaphone.dm

```diff
@@ -46,7 +46,7 @@
 				continue
 			listener.show_message("<B>[user]</B> broadcasts, [FONT_SIZE_LARGE("\"[message]\"")]", SHOW_MESSAGE_AUDIBLE) // 2 stands for hearable message
 			langchat_long_listeners += listener
-		playsound(loc, 'sound/items/megaphone.ogg', 100, FALSE, TRUE)
+		//playsound(loc, 'sound/items/megaphone.ogg', 100, FALSE, TRUE) // Stops playing. Commenting out so we can just plop a new sound in.
```

Comment:
if you're going to remove something, do it fully.

no comment on if the PR is good or bad